lib/modules/datasource/aws-eks-addon/index.spec.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  type AddonInfo,
+  type AddonVersionInfo,
+  DescribeAddonVersionsCommand,
+  DescribeAddonVersionsResponse,
+  EKSClient,
+} from '@aws-sdk/client-eks';
+import { mockClient } from 'aws-sdk-client-mock';
+import { getPkgReleases } from '..';
+import { AwsEKSAddonDataSource } from '.';
+
+const datasource = AwsEKSAddonDataSource.id;
+const eksMock = mockClient(EKSClient);
+
+const addonVersion1: AddonVersionInfo = {
```

Comment:
please inline those variables to appropriate scope, that are all only used once. (perf)